src/mavlink-router/endpoint.cpp

```diff
@@ -881,7 +881,11 @@ int UdpEndpoint::open(const char *ip, unsigned long port, bool to_bind)
     } else {
 #endif
     sockaddr.sin_family = AF_INET;
-    sockaddr.sin_addr.s_addr = inet_addr(ip);
+    struct hostent *server;
+    server = gethostbyname(ip);
```

Comment:
Isn't that [obsolete](https://www.man7.org/linux/man-pages/man3/gethostbyname.3.html)? :thinking: 

> The gethostbyname*(), gethostbyaddr*(), herror(), and hstrerror()
       functions are obsolete.  Applications should use [getaddrinfo(3)](https://www.man7.org/linux/man-pages/man3/getaddrinfo.3.html),
       [getnameinfo(3)](https://www.man7.org/linux/man-pages/man3/getnameinfo.3.html), and [gai_strerror(3)](https://www.man7.org/linux/man-pages/man3/gai_strerror.3.html) instead.